### ISSUE TITLE
High-core-count macOS 10.13 VM becomes unusable when building Octave.app

### ISSUE THREAD

**Issue by apjanke** [octave-app/octave-app#178]
I bumped up hicks, one of my macOS 10.13 VMs running in VMware Fusion Pro 11.5 on 10-core iMac Pro angharad running macOS 10.14 host OS, to 16 virtual cores to try to get it to build Octave.app faster. The VM becomes unusable once into the build, though: I can come back a few hours after kicking off the build, and the VM will still be at about 500% CPU usage in Activity Monitor, and it will be unusably unresponsive: it will take several seconds to get from the screen saver to the password entry screen, the password entry screen will take several seconds to echo each character typed in, and will revert back to the screen saver before the password is fully entered. I think this is something besides just the VM being busy during the build, because I've let it go long enough that I'd expect the build to be long finished.

**Comment by apjanke** [comment]
I bumped hicks down to 8 virtual cores and am doing another overnight build on it.

**Comment by apjanke** [comment]
Don't know if this worked; the overnight build failed because ImageMagick went 404 yesterday. Waiting on an upstream fix from Homebrew. https://github.com/Homebrew/homebrew-core/pull/45869

**Comment by apjanke** [comment]
It worked. ImageMagick got fixed upstream, and a rebuild succeeded.

Maybe macOS 10.13 VMs under vmWare are just wonky when you run them with more than 8 vCores. That wouldn't be too surprising, since Macs with more than 4 hardware cores are kind of rare, so wouldn't see as much testing.

**Comment by apjanke** [comment]
I don't think there's anything we can do about this. Closing as upstream/wontfix; just something we'll have to live with and watch out for.

**Comment by apjanke** [comment]
Reported upstream to VMware; Support Request Confirmation Number 20102127902.

**Comment by apjanke** [comment]
Oh my gosh, I'm a moron: when I bumped up the VM's core count, and thus the parallelism of the build process, I left it with 4 GB RAM. It was probably just running out of memory and thrashing on the swap file. I bumped it up to 14 vcores and 16 GB RAM today, and it worked just fine.

**Comment by apjanke** [comment]
Removed the "upstream" label because this wasn't a VMware problem; it was just user error.
